src/server/utils/getTranslations.js

```diff
@@ -0,0 +1,24 @@
+const mergeDeep = require('merge-deep')
+
+const SUPPORTED_LOCALES = {
+  EN: 'en'
+}
+
+const getTranslations = ({ locale }) => {
+  const defaultTranslations = require('translations/en.json')
+  let localeTranslations
+  switch (locale) {
+    case SUPPORTED_LOCALES.EN:
+    default:
+      localeTranslations = null
+      break
+  }
+
+  const finalTranslations = localeTranslations
+    ? mergeDeep(defaultTranslations, localeTranslations)
+    : defaultTranslations
```

Comment:
This uses english translations as the default. I think ideally these merges would be pre-compiled. I'll take care of this before merging